src/main/java/uk/gov/hmcts/reform/hmc/api/controllers/HearingsController.java

```diff
@@ -126,6 +127,44 @@ public ResponseEntity<Object> getHearingsByCaseRefNo(
         }
     }
 
+    /**
+     * End point to fetch all the hearings which belongs to a particular caseRefNumber.
+     *
+     * @return caseHearingsResponse, all the hearings which belongs to a particular caseRefNumber.
+     * @header serviceAuthorization, S2S authorization token.
+     * @header caseReference, CaseRefNumber to take all the hearings belongs to this case.
+     */
+    @GetMapping(path = "/hearings-by-list-of-cases")
+    @ApiOperation("get hearings by case reference number")
+    @ApiResponses(
+            value = {
+                @ApiResponse(code = 200, message = "get hearings by caseRefNo successfully"),
+                @ApiResponse(code = 400, message = "Bad Request")
+            })
+    public ResponseEntity<Object> getHearingsByListOfCaseRefNos(
+            @RequestHeader(AUTHORIZATION) String authorization,
+            @RequestHeader(SERVICE_AUTHORIZATION) String serviceAuthorization,
+            @RequestBody final Cases cases) {
```

Comment:
Can we not have List<String> caseIds instead of Cases object?

Reply:
Yes ,it can be and have modified accordingly